internal/datastore/spanner/options.go

```diff
@@ -16,6 +16,22 @@ type spannerOptions struct {
 	credentialsFilePath         string
 	emulatorHost                string
 	disableStats                bool
+
+	migrationPhase string
+}
+
+type migrationPhase uint8
+
+const (
+	writeChangelogReadChangelog migrationPhase = iota
+	writeChangelogReadStream
+	complete
```

Comment:
I think when I did this for postgres, I made the zero value for the migrationPhase be complete, so the lack of a value would be the end migration state and it would make it easier for people who had fully migrated to stop passing the value.

Reply:
I actually went back and copied this directly from the 1.14 code and then modified it. I'm fine switching up the values, but since we always read the string value, I think its fine

Reply:
Yep, I see now it's the map that converts the zero value of the string to the `complete` phase.